### ISSUE TITLE
new install in docker does not start - missing /var/run/gerbera/.config/gerbera

### ISSUE THREAD

**Issue by hmoffatt** [gerbera/gerbera#1495]
I'm trying to run gerbera/gerbera:latest in docker using the instructions in the README, but it doesn't start, with the error:

```
2021-06-01 04:40:29  error: [/gerbera_build/src/config/config_manager.cc:164] load Error: Directory '/var/run/gerbera/.config/gerbera' does not exist
```

That directory seems to be <home> in the default generated config file but nothing creates it.

**Comment by hmoffatt** [comment]
I edited the home directory to a directory that has a volume mounted and then it starts OK.

**Comment by KarlStraussberger** [comment]
Maybe you can provide a PR to update the Readme. Help is always welcome.

**Comment by hmoffatt** [comment]
This is a bug in the config generation or the dockerfile rather than the instructions.

**Comment by KarlStraussberger** [comment]
If you provide `/var/run/gerbera` as a volume and supply the right structure from your host?

**Comment by hmoffatt** [comment]
Yes, that does work, thanks. I think the docker-entrypoint.sh should make that directory after it generates the initial config file - what do you think?

**Comment by KarlStraussberger** [comment]
Sounds reasonable. Should be as easy to use as possible.

**Comment by KarlStraussberger** [comment]
Found the issue in the auto-generated config.

It's fixed in #1529

**Comment by hmoffatt** [comment]
Thanks, it works with a brand new config directory.

The change to the entrypoint script to make an old config work doesn't work though:
```mkdir /var/run/gerbera/.config/gerbera```
It needs mkdir -p because the .config subdirectory doesn't exist.
